include/crocoddyl/multibody/frames.hpp

```diff
@@ -136,6 +136,40 @@ struct FrameFrictionConeTpl {
   FrictionCone oRf;
 };
 
+template <typename _Scalar>
+struct FrameCoPSupportTpl {
+  EIGEN_MAKE_ALIGNED_OPERATOR_NEW
+
+  typedef _Scalar Scalar;
+  typedef typename MathBaseTpl<Scalar>::Vector2s Vector2s;
+  typedef Eigen::Matrix<Scalar, 4, 6> Matrix46;
+
+  explicit FrameCoPSupportTpl() : frame(0), support_region(Vector2s::Zero()) {}
+  FrameCoPSupportTpl(const FrameCoPSupportTpl<Scalar>& value) : frame(value.frame), support_region(value.support_region) {}
+  FrameCoPSupportTpl(const FrameIndex& frame, const Vector2s& support_region) : frame(frame), support_region(support_region) {}
+  friend std::ostream& operator<<(std::ostream& os, const FrameCoPSupportTpl<Scalar>& X) {
+    os << "      frame: " << X.frame << std::endl << "foot dimensions: " << std::endl << X.support_region << std::endl;
+    return os;
+  }
+
+  // Define the inequality matrix A to implement A * f <= 0 compare eq.(18-19) in https://hal.archives-ouvertes.fr/hal-02108449/document
+  //Matrix3s c_R_o = Quaternions::FromTwoVectors(nsurf_, Vector3s::UnitZ()).toRotationMatrix(); TODO: Rotation necessary for each row of A?
+  void update_A() {
+  A_ << 0, 0, support_region[0] / 2, 0, -1, 0,
+        0, 0, support_region[0] / 2, 0, 1, 0,
+        0, 0, support_region[1] / 2, 1, 0, 0,
+        0, 0, support_region[1] / 2, -1, 0, 0;
+  }
+
+  const Matrix46& get_A() const {
+    return A_;
+}
+
+  FrameIndex frame; //!< name of the contact frame 
+  Vector2s support_region; //!< dimension of the foot surface dim = (length, width)
+  Matrix46 A_; //!< inequality matrix
```

Comment:
You also forgot to bind this functions. Additionally, it needs to be renamed as `A`, not `A_`.

Quick clarification about the code style. Structure's members do not use `_` as last character, the opposite is for the class's members. (this is Google code style that we follow in the project)